src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculations = {
+    result: 0,
+    add(n) {
+      return calculations.result + n;
+    },
```

Comment:
Better not use calculation.result, because the variable name can change and ruin everything. Consider accepting result as a parameter.